code/modules/paperwork/filingcabinet.dm

```diff
@@ -86,9 +86,7 @@
 			usr.put_in_hands(P)
 			updateUsrDialog()
 			icon_state = "[initial(icon_state)]-open"
-			spawn(0)
-				sleep(5)
-				icon_state = initial(icon_state)
+			addtimer(VARSET_CALLBACK(src, icon_state, initial(icon_state)), 5)
```

Comment:
```suggestion
			addtimer(VARSET_CALLBACK(src, icon_state, initial(icon_state)), 0.5 SECONDS)
```

Reply:
assuming this is an animation length

Reply:
The animation uses the same time metric, no?